### PR TITLE
pyscroll: init at 2.19.2

### PR DESCRIPTION
**Pull request by oxzi**
###### Motivation for this change
Add the [pyscroll](https://github.com/bitcraft/pyscroll) library

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: pkgs/development/python-modules/pyscroll/default.nix [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  version = "2.19.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "175d39rymnx3v3mcfjjkr1wv76nsl1s00z04nzsspklyk0ih2783";
+  };
+
+  propagatedBuildInputs = [ pygame ];
+
+  meta = {
```
Comment (jonringer) [inline pkgs/development/python-modules/pyscroll/default.nix]:
```suggestion
  meta = with lib; {
```

File: pkgs/development/python-modules/pyscroll/default.nix [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    homepage = "https://github.com/bitcraft/pyscroll";
+    description = "Python library for scrolling games with animated maps in pygame";
+    license = lib.licenses.lgpl3;
```
Comment (jonringer) [inline pkgs/development/python-modules/pyscroll/default.nix]:
```suggestion
    license = licenses.lgpl3;
```

File: pkgs/development/python-modules/pyscroll/default.nix [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    maintainers = with lib.maintainers; [ geistesk ];
```
Comment (jonringer) [inline pkgs/development/python-modules/pyscroll/default.nix]:
```suggestion
    maintainers = with maintainers; [ geistesk ];
```

**Review by jonringer — COMMENTED** [review]
please pull from github so that we can have some tests, just unittests are good  :)

Unfortunately you'll have to match the date to a commit to figure out the version github, as they didn't use a tag/release mechanism there.

**Comment by jonringer** [comment]
I would focus on just unit tests, if they make the distinction.

**Comment by geistesk** [comment]
I have opened an issue regarding the failing tests in the upstream repository https://github.com/bitcraft/pyscroll/issues/34

**Review by FRidh — APPROVED** [review]
Looks good, merge conflict needs to be resolved though.

**Review by FRidh — CHANGES_REQUESTED** [review]
Ah, right, the tests...

**Comment by mweinelt** [comment]
Seems like the upstream has put work into the tests. Can you give it another try @geistesk?

**Pull request closed without merging**